core/src/main/java/nl/nn/adapterframework/senders/XmlValidatorSender.java

```diff
@@ -66,12 +67,12 @@ public ParameterList getParameterList() {
 	public Message sendMessage(Message message, PipeLineSession session) throws SenderException, TimeoutException {
 		String fullReasons="tja";
 		try {
-			String resultEvent = validate(message, session, getLogPrefix(),null,null);
+			ValidationResult validationResult = validate(message, session, getLogPrefix(),null,null);
 			
-			if (AbstractXmlValidator.XML_VALIDATOR_VALID_MONITOR_EVENT.equals(resultEvent)) {
+			if (validationResult == ValidationResult.VALID || validationResult == ValidationResult.VALID_WITH_WARNINGS) {
```

Comment:
Moet deze ook door als hij valid with warnings is, of alleen als recoverFromRecoverableErrors is gezet?